TestingTool/scenario.py

```diff
@@ -4,7 +4,26 @@
 from mongoengine import StringField, EmbeddedDocumentListField
 from .task import Task
 from .result import Result
+from selenium import webdriver
 
 
 class Scenario(EmbeddedDocument):
-    pass  # I'm just a placeholder
+    name = StringField(max_length=128, required=True)
+    url = StringField(max_length=512, required=True)
+    tasks = EmbeddedDocumentListField(Task)
+
+    def run(self):
+        driver = webdriver.Firefox()
+        driver.implicity_wait(5)
```

Comment:
Nu am putut sa gasesc o explicatie asupra efectului acestei functii asupra driverului, intrucat pare ca e folosita doar in momentul in care vrem sa setam un timeout, ceea ce nu se intampla aici. Daca ai o explicatie pentru folosirea ei, as vrea sa imi scrii.